setup.cfg

```diff
@@ -1,24 +1,3 @@
-[metadata]
```

Comment:
This package seems to be trying to declare its build metadata in two places. If you build the package, the stuff in `pyproject.toml` is actually being used and this is ignored. I'm deleting this. My guess is that at some point in the past `setup.cfg` was the source of truth then someone migrated to `pyproject.toml` but didn't clean up?